os_sys/rm.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+
+import argparse
+import os.path
+import sys
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description="Remove each specified file. By default, it does not remove "
+                                                 "directories.")
+    parser.add_argument('-r', '--recursive', action='store_true', default=False,
+                        help='Remove directories and their contents recursively.')
+    parser.add_argument('entities_to_delete', nargs='+', default=sys.stdin)
```

Comment:
default тут никогда не будет использован, так как `nargs="+"`
```suggestion
    parser.add_argument('entities_to_delete', nargs='+')
```